src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -584,6 +582,25 @@ protected function get_views() {
 						$count
 					);
 					break;
+				default:
+					/**
+					 * Filter the status text of default switch case in the Plugins list table.
+					 *
+					 * @since 6.9.0
+					 *
+					 * @param string $text   Plugins list status text. Default empty string.
+					 * @param int    $count  Count the Number of plugins.
+					 * @param string $type   The status slug being filtered.
+					 */
+					$text = apply_filters( 'plugins_list_status_text', '', $count, $type );
```

Comment:
Instead of checking for a string as suggested below, should we force a string value from the filter? 

```suggestion
					$text = (string) apply_filters( 'plugins_list_status_text', '', $count, $type );
```

Reply:
That could result in a PHP warning if the filter returns an array: https://3v4l.org/XAR5P

Reply:
True. My thought was that, since WP core does this kind of casting already in numerous places, it would be safe to do since a warning doesn't stop execution, and it provides clues about incorrect code that doesn't provide expected results, whereas without it we kinda hide the problem. But as your example shows, having a string that says "Array" isn't very helpful either. Happy to withdraw my suggestion.

Reply:
I don't feel strongly about it. If someone uses this filter to return anything other than a string then they are doing it wrong anyway. There's probably a literal billion other places on WP where this kind of thing would also happen currently.